.github/ISSUE_TEMPLATE/new_component_request.md

```diff
@@ -0,0 +1,37 @@
+---
+name: New component request
+about: Suggest a component for this project
+title: 'feat: add [component name] component'
+labels: enhancement 🚀
+---
+
+<!-- 
+    Before submitting an issue, please consult our [documentation](https://decathlon.design). 
+    Please make sure you are posting an issue pertaining to the Decathlon Design System. 
+
+    Note that before opening an issue, you can chat maintainer of this project on channel
+    #vitamin-ios in [Decathlon Design System's Slack(https://join.slack.com/t/decathlon-design/shared_invite/zt-ou0n9qas-n_oamDSVUIqvLqNO1LETJg).
+
+    If you want to know how to contribute to this project, you can check our CONTRIBUTING file:
+    https://github.com/Decathlon/vitamin-ios/blob/main/CONTRIBUTING.md
+
+    If a section isn't adapted for your request, please remove it to avoid any unnecessary section.
+
+    Thanks!
+-->
+
+**New component**
+<!-- A clear and concise description of what the feature request is. Please include if your feature request is related to a problem. -->
+
+**Documentation links**
+
+- [Documentation](https://decathlon.design/...)
+- [Figma](https://figma.com/...)
+
+**Checklist**
+<!-- Please, don't modify this checklist. -->
+
+- [ ] CSS
```

Comment:
I do not know what to write here instead of that, but I do not think JS frameworks belong to this repo ;)

Reply:
ahah you're right, fixed :)